### ISSUE TITLE
[v4] `absolute` utility class not working

### ISSUE THREAD

**Issue by emmanuelchucks** [marklawlor/nativewind#577]
**Describe the bug**
Trying out the v4 alpha

```tsx
// this doesn't work
<View className="absolute bottom-0 right-0" />

// but this works
<View style={{ position: "absolute" }} className="bottom-0 right-0" />
```

**To Reproduce**
1. Install v4 alpha
2. Use `absolute` class name on any element
3. Element is not positioned absolutely

**Expected behavior**
`absolute` class should position element absolutely.

**Expo Snack**
I don't think v4 works on Snack yet. If I'm wrong let me know how to get it working.

**Comment by marklawlor** [comment]
Fixed in `4.0.0-alpha.23`

**Comment by emmanuelchucks** [comment]
Bug is still present unfortunately

**Comment by emmanuelchucks** [comment]
Fixed in 4.0.0-alpha.25

Thanks for your hard work.

**Comment by AyoCodess** [comment]
this bug is back
